serverLib.sml

```diff
@@ -964,7 +964,8 @@ in
             val files =
               List.map (fn id => OS.Path.concat("running",Int.toString id)) (running()) @
               List.map (fn id => OS.Path.concat("finished",Int.toString id)) (finished())
-            val (t,fs) = timings_of_dir dir files
+            val recent_files = take 100 (List.rev files)
```

Comment:
I don't think these are necessarily in recency order are they? Maybe you'd need to sort by job id?

Reply:
Ah yes sorry, `running()` and `finished()` are only separately in order of ascending job ID. Perhaps a simple fix is to use only `finished()`, particularly because there are rarely more than a couple of `running()` ones. What do you think?

Reply:
I believe they are sorted in ascending order of job ID, because [`finished`](https://github.com/CakeML/regression/blob/master/serverLib.sml#L235) is defined in terms of `read_list`, which in its [loop body](https://github.com/CakeML/regression/blob/master/serverLib.sml#L208) inserts each job ID in order (using [`insert`](https://github.com/CakeML/regression/blob/master/utilLib.sml#L11)).

Reply:
Amazing - thanks for finding that.

Reply:
Looks like it'll be sorted descending? But that's what we want.

Reply:
Ah sorry, I got confused with `waiting()` which is in ascending order - so I believe the `List.rev` when defining `recent_files` is incorrectly giving us the oldest jobs (and this seems to track with the live regression webpage, which doesn't show expected times for newer examples). Sorry for the bug here - I suppose we need to delete `List.rev`. I don't know if you'd prefer to commit this directly or to receive another small PR?

Reply:
Fixed in 1102c51acb44e07464eeff3bf830f4ae4308d30b